sechub-schedule/src/main/java/com/mercedesbenz/sechub/domain/schedule/job/SecHubJobInfoForUserService.java

```diff
@@ -0,0 +1,123 @@
+package com.mercedesbenz.sechub.domain.schedule.job;
+
+import javax.annotation.PostConstruct;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.data.domain.Example;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.PageRequest;
+import org.springframework.data.domain.Pageable;
+import org.springframework.data.domain.Sort;
+import org.springframework.data.domain.Sort.Direction;
+import org.springframework.stereotype.Service;
+
+import com.mercedesbenz.sechub.domain.schedule.ScheduleAssertService;
+import com.mercedesbenz.sechub.sharedkernel.MustBeDocumented;
+import com.mercedesbenz.sechub.sharedkernel.Step;
+import com.mercedesbenz.sechub.sharedkernel.usecases.job.UseCaseUserListsJobsForProject;
+
+@Service
+public class SecHubJobInfoForUserService {
+
+    private static final Logger LOG = LoggerFactory.getLogger(SecHubJobInfoForUserService.class);
+
+    private static final int DEFAULT_MAXIMUM_LIMIT = 100;
+    private static final int DEFAULT_MAXIMUM_PAGE = 100;
+    private static final int MINIMUM_SIZE = 1;
+
+    private static final int MINIMUM_PAGE = 0;;
+
+    @Autowired
+    SecHubJobRepository jobRepository;
+
+    @Autowired
+    ScheduleAssertService assertService;
+
+    @Value("${sechub.project.joblist.size.max:" + DEFAULT_MAXIMUM_LIMIT + "}")
+    @MustBeDocumented("Maximum limit for job information list entries per page")
+    int maximumSize = DEFAULT_MAXIMUM_LIMIT;
+
+    @Value("${sechub.project.joblist.page.max:" + DEFAULT_MAXIMUM_PAGE + "}")
+    @MustBeDocumented
+    int maximumPage = DEFAULT_MAXIMUM_PAGE;
+
+    @PostConstruct
+    void postConstruct() {
+        if (maximumSize < MINIMUM_SIZE) {
+            LOG.warn("Illegal maximum size defined: {} - will use: {} as fallback.", maximumSize, DEFAULT_MAXIMUM_LIMIT);
+            maximumSize = DEFAULT_MAXIMUM_LIMIT;
+        }
+        if (maximumPage < MINIMUM_PAGE) {
+            LOG.warn("Illegal maximum page defined: {} - will use: {} as fallback.", maximumPage, DEFAULT_MAXIMUM_PAGE);
+            maximumPage = DEFAULT_MAXIMUM_PAGE;
+        }
+    }
+
+    @UseCaseUserListsJobsForProject(@Step(number = 2, name = "Assert access by service and fetch job informaiton for user"))
```

Comment:
Found the typo while looking at the generated documentation.
```suggestion
    @UseCaseUserListsJobsForProject(@Step(number = 2, name = "Assert access by service and fetch job information for user"))
```